### PR TITLE
Fix/animation time zero issue

### PR DESCRIPTION
**Pull request by z-shaffer**
Resolves issue #1222

### CODE REVIEW

File: application/testing/CMakeLists.txt [hunk 1]
```diff
@@ -541,6 +541,12 @@ if(F3D_PLUGIN_BUILD_EXODUS)
   # Test no render animation time. Regex contains a part of the range of the ACCL field.
   f3d_test(NAME TestNoRenderAnimation DATA small.ex2 ARGS  --load-plugins=exodus --animation-time=0.003 REGEXP "-521950, 6.57485" NO_RENDER)
 
+  # Test if the animation-time works when set to zero and time range[0] is less than zero
+  f3d_test(NAME TestTimeRangeLessThanZero DATA negative_range_animated.e ARGS -sgxn --dry-run --animation-time=0 DEFAULT_LIGHTS)
```
Comment (mwestphal) [inline application/testing/CMakeLists.txt:545]: --dry-run should not be needed here

Comment (mwestphal) [reply application/testing/CMakeLists.txt:545]: also remove `-gxn` no need to display them here. keep the `s` though.

Remark is for both tests

**Review by mwestphal — CHANGES_REQUESTED** [review]
small changes needed

File: application/testing/CMakeLists.txt (hunk 1 above)
Comment (mwestphal) [reply application/testing/CMakeLists.txt:545]: also needs `--load-plugins=exodus`

Comment (z-shaffer) [reply application/testing/CMakeLists.txt]: done

**Review by mwestphal — APPROVED** [review]

**Pull request merged**